futurex_openedx_extensions/dashboard/views.py

```diff
@@ -203,8 +207,9 @@ def get_queryset(self) -> QuerySet:
 
 class CoursesView(ListAPIView, FXViewRoleInfoMixin):
     """View to get the list of courses"""
-    serializer_class = serializers.CourseDetailsSerializer
+    authentication_classes = [SessionAuthentication, BearerAuthentication]
```

Comment:
Please make a constant, to reduce git diff.

If it's a setting, it has to be import path rather than actual class:

FX_AUTHENTICATION_CLASSES = [
  'futurex-openedx-extensions.auth.BearerAuthentication',
  'another_one_here',
]

Reply:
done